src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+
+  --link-color: #00acdc;
+  --header-height: 60px;
+
+  font-family: Roboto, Arial, sans-serif;
+}
+
+ul {
+  list-style: none;
```

Comment:
Avoid using tag names for styling. You should use classes instead. This is a CSS style issue. You should style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.